src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,13 @@
-export const UserInfo = () => {};
+import { User } from '../../types/User';
+
+type Props = {
+  user: User
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  return (
+    <a className="UserInfo" href={user.email}>
```

Comment:
Here's what I see when I click on this element, use `mailto:` link
![image](https://user-images.githubusercontent.com/37161315/180200378-b22b44c2-08a1-4daa-9d28-83e1fd6e3283.png)